modules/ossm-troubleshooting-proxy.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/-ossm-troubleshooting-istio.adoc
+
+[id="ossm-troubleshooting-proxy_{context}"]
+= Troubleshooting Envoy proxy
+
+The Envoy proxy intercepts all inbound and outbound traffic for all services in the service mesh. Envoy also collects and reports telemetry on the service mesh. Envoy is deployed as a sidecar to the relevant service in the same pod.
+
+== Enabling access logs
+
+Envoy access logs useful in diagnosing traffic failures and flows and help with end-to-end traffic flow analysis.
+
+To enable access logging for all istio-proxy containers, edit the ServiceMeshControlPlane (SMCP) file.
+
+[source,terminal]
+----
+$ oc edit smcp <SMCP-name>
+----
+
+[source,yaml]
+----
+spec:
+  proxy:
+    accessLogging:
+      file:
+        name: /dev/stdout     #file name
+----
+
+
+== Common Envoy error messages
+Error message:
+`2020-08-22T15:06:29.105313Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 1 successful, 0 rejected; lds updates: 0 successful, 0 rejected`
+
```

Comment:
For some reason (looking at the preview), this message is partially formatted.

Reply:
Fixed a couple of formatting issues.